frontend/rust-lib/flowy-database2/src/services/database/database_editor.rs

```diff
@@ -1854,6 +1854,69 @@ impl DatabaseEditor {
     }
   }
 
+  pub async fn get_prompts_from_database(
+    &self,
+    view_id: &str,
+  ) -> Result<Vec<CustomPromptPB>, FlowyError> {
+    let fields = self.get_fields(view_id, None).await;
+    let primary_field = fields
+      .iter()
+      .find(|f| f.is_primary)
+      .ok_or(FlowyError::internal().with_context("Cannot find primary field"))?;
+    let content_field = fields
+      .iter()
+      .find(|f| f.name == "Content" && f.field_type == FieldType::RichText.value())
```

Comment:
do we need to use `to_lowercase` here as well?

Reply:
Yes